charts/bindplane/templates/bindplane-jobs.yaml

```diff
@@ -0,0 +1,447 @@
+{{- if eq (include "bindplane.deployment_type" .) "Deployment" }}
+apiVersion: apps/v1
+kind: Deployment
+metadata:
+  name: {{ include "bindplane.fullname" . }}-jobs
+  namespace: {{ .Release.Namespace }}
+  labels:
+    app.kubernetes.io/name: {{ include "bindplane.name" . }}
+    app.kubernetes.io/stack: bindplane
+    app.kubernetes.io/component: jobs
+    app.kubernetes.io/instance: {{ .Release.Name }}
+    app.kubernetes.io/managed-by: {{ .Release.Service }}
+spec:
+  replicas: 1
```

Comment:
Unlike bindplane.yaml, we always set replicas to 1 to ensure there is a single jobs pod.

During a rolling update, it is okay if multiple job pods exists during the lifetime of the pod rollout.